package by.training.task1.bean.exception;